api.go

```diff
@@ -33,9 +28,19 @@ func main() {
 	r.StrictSlash(false)
 
 	r.HandleFunc("/ping", http.HandlerFunc(handlers.Ping)).Methods("GET")
+
 	r.HandleFunc("/", http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
-		rw.Write(welcome)
+		if bypassCaptcha {
+			http.ServeFile(rw, r, "./www/bypass.html")
```

Comment:
Why creaing another template and duplicating the html?. Golang templates allow to add `if` statements, we can stick to just one welcome.html and check for the `bypassCaptcha there`